telehealth-example/src/main/java/com/pubnub/components/example/telehealth/ui/view/Channel.kt

```diff
@@ -71,26 +70,23 @@ object Channel {
         val channels = remember {
             channelViewModel.getAll(transform = {
                 map { channelUi: ChannelUi ->
-                    channelUiMapper.map(channelUi, chatParameters.userId)
+                    channelUiMapper.map(channelUi, userParameters.userId)
                 }
             })
         }
-        val context = LocalContext.current
         CompositionLocalProvider {
             Content(
                 channels = channels,
                 onSelected = {
-                    val intent = Intent(context, ChatActivity::class.java).apply {
-                        putExtra(PARAMETERS_BUNDLE_KEY,
-                            chatParameters.copy(
-                                channelId = it.id,
-                                secondUserId = it.description ?: "",
-                                secondUserName = it.name,
-                            ))
-                    }
-                    context.startActivity(intent)
+                    onChannelSelected(ChatParameters(
+                        userId = userParameters.userId,
+                        type = userParameters.type,
+                        channelId = it.id,
+                        secondUserId = it.description ?: "",
```

Comment:
Do we really need to pass user ids to callback? Shouldn't it be obtained from Channel data?